src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -2833,6 +2833,25 @@ skip:
   }
   //------------------------------------------------------------------------------------------------------------------------
   template<class t_core>
+  void t_cryptonote_protocol_handler<t_core>::on_connection_new(cryptonote_connection_context &context)
+  {
+    context.set_max_bytes(nodetool::COMMAND_HANDSHAKE_T<cryptonote::CORE_SYNC_DATA>::ID, 65536);
```

Comment:
This group of values - the ping and request flags in particular - seem pretty aggressive and might make expansion complicated. Although it seems unlikely that they get changed at all.